eventlet/support/greendns.py

```diff
@@ -179,11 +179,16 @@ def _readlines(self):
         Note that this performs disk I/O so can be blocking.
         """
         lines = []
+        if sys.version_info < (3, 4):
+            mode = 'rU'
+        else:
+            mode = 'r'
```

Comment:
Again. `open(..., 'rb')` Always work explicitly with either bytes or strings. Files are obviously bytes.

It seems possible to read a text file without checking Python version.